### ISSUE TITLE
Need to update badgr server email text (remove example.com)

### ISSUE THREAD

**Issue by lidijarad** [proversity-org/badgr-server#6]
Hello from example.com!

You're receiving this email because you or someone else has requested a password for your user account at example.com. It can be safely ignored if you did not request this reset. If this is you and you're ready to change your password, click the button below.

Thank you for using example.com!

**Comment by lidijarad** [comment]
@nicovanniekerk @jagonzalr

**Comment by nicovanniekerk** [comment]
@lidijarad if this was done, please close the ticket, if not i am assigning it to myself, so if its still here in a few days ill update this :)

**Comment by lidijarad** [comment]
@nicovanniekerk, yes please. I do not know if badgr is going to be client facing but I assume we need to update the email text.
